core/src/main/resources/bosatsu/predef.bosatsu

```diff
@@ -42,6 +42,9 @@ export [
   trace,
   uncurry2,
   uncurry3,
+  markdown,
+  header,
+  list_vis,
```

Comment:
small comment, I'd really like to keep the predef as minimal as possible (even now I think it could be too big).
 
I think these functions should be in a separate module (which could be configured to be auto imported by your notebook system).

Reply:
I think I just struggled how to have a second .bosatsu file make it into the build. But I should definitely go back and fix that.